src/index.js

```diff
@@ -1,10 +1,34 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { createStore } from 'redux';
+import { Provider } from 'react-redux';
+import rootReducer from './reducers'
+
 import './index.css';
-import App from './App';
+import App from './containers/App';
 import * as serviceWorker from './serviceWorker';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+
+const generateID = () => {
+  return Math.floor((Math.random() * 1000) + 1);
+}
+
+const INITIAL_STATE = {
+  books: [
+    { id: generateID(), title: "Laws of Power", category: "Learning" },
+    { id: generateID(), title: "Why Men want Sex and Women want Love", category: "Learning" },
+    { id: generateID(), title: "Life 3.0", category: "Sci-Fi" }
+  ],
+  filter: 'All'
+}
+
+const store = createStore(rootReducer, INITIAL_STATE);
```

Comment:
It is very helpful to use the Redux devtools plugin. Kindly see [this](https://github.com/zalmoxisus/redux-devtools-extension) page to learn how to install it in your browser and here in the project.